### DISCUSSION TITLE
Group 3 Project successfully submitted to Canvas Sunday July 31, 2022

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#26] [Announcements]
Hey Team, 

I was able to successfully submit the group3 project via canvas.   I think that is a wrap! 👍 

I wanted to personally thank you both- Taylor and Shannon for a job well done!   It was such a pleasure working with you both.

I wish you the best in the completion of the rest of your coursework, and in life in general!   

Cheers!
-Matt Kopay

**Comment by shannonkuhn92** [comment]
Awesome. Same to you, Matt (and Taylor)! Having such a great team made this project so much less stressful. Thanks and good luck out there! :)

**Comment by tay-baum** [comment]
I can’t say it better so I’ll just say thank you both! I hope the end of the semester goes well and you get a good break in before fall! All the best!
